Marketplace workflow job failed when tagging v0.103.0

### Description

The marketplace workflow [job](https://github.com/hashgraph/hedera-mirror-node/actions/runs/8803280115/job/24162324042) failed in its `Build images` step when tagging v0.103.0 release, 

```
+ docker pull bats/bats:v1.11.0 --platform linux/amd64
Error response from daemon: manifest for bats/bats:v1.11.0 not found: manifest unknown: manifest unknown
Error: Process completed with exit code 1.
```

### Steps to reproduce

check the referenced workflow job run log

### Additional context

_No response_

### Hedera network

other

### Version

v0.103.0

### Operating system

None

--- a/charts/marketplace/gcp/release.sh
+++ b/charts/marketplace/gcp/release.sh
@@ -20,7 +20,7 @@ fi
 
 target_tag="${target_tag#v}" # Strip v prefix if present
 target_tag_minor="${target_tag%\.*}"
-bats_tag="v1.11.0"
+bats_tag="1.11.0"
 postgresql_tag="14.11.0-debian-12-r8"
 registry="gcr.io/mirror-node-public/hedera-mirror-node"
 

